### ISSUE TITLE
Client secret is not 27 characters

### ISSUE THREAD

**Issue by dsychin** [not-an-aardvark/reddit-oauth-helper#18]
I am not able to use the cli because it is saying that the client secret should be 27 characters, but mine is 30 characters.
```
Please enter your client secret. This is the 27-character string that appears when you click "edit" next to your app on this page: https://www.reddit.com/prefs/apps
```

**Comment by hemisemidemipresent** [comment]
having the same issue, tried making a new app but the client secret is also 30 characters long. Deleting the last 3 characters do not work, deleting the first 3 characters doesn't work either. Is there any fix currently?

**Comment by dsychin** [comment]
The only way for me to get it to work is to use the web version at https://not-an-aardvark.github.io/reddit-oauth-helper/

**Comment by GalaxySH** [comment]
Same issue, this makes the helper unusable

**Comment by EMenardc** [comment]
Yep, having the same issue..

I went into the index.js and changed the check to 30 characters, it seems to fix the issue.

**Comment by ahsan-a** [comment]
it appears to be 30 characters in the code, but using the `npx reddit-oauth-helper` command seems to not have been updated. I just cloned the repo, installed the npm modules and ran from my computer

**Comment by not-an-aardvark** [comment]
Thanks for the reminder, I just published v0.3.1 so this should be fixed now.
